wordpress-mobile/WordPress-iOS
# Improve internal icon generation

This PR modifies the script that adds the version number to our icons when building for internal releases, so that the icons are not modified in the working directory.

This PR:

- Renames the relevant build phase to "App Icons: Add Version For Internal Releases".
- Moves the phase one step down (needed so that the icons are copied to the intermediate directories before it's run).
- Moves the script to an external file to make it easier to edit and maintain.
- Improves the script so that it can deal with paths with spaces (was breaking).
- This PR also uses the long version number instead of the short one, so we can have a short version number in internal builds too.

**How to test:**

1. Archive an internal release.
2. Make sure the icons in the final product are right

**How to check the icons in the final archive:**

 - Open the organizer
 - Right click on .xcarchive and select "Show in Finder"
 - Go to Products > Applications
 - Right click on "WordPress.App" and select "Show Package Contents".
 - Verify the icons are OK.
